src/main/java/edu/harvard/iq/dataverse/EditDatafilesPage.java

```diff
@@ -3044,7 +3048,7 @@ public void saveAdvancedOptions() {
     }
 
     public boolean rsyncUploadSupported() {
-        // ToDo - rsync was written before multiple store support and currently is hardcoded to use the "s3" store. 
+        // ToDo - rsync was written before multiple store support and currently is hardcoded to use the DataAccess.S3 store. 
```

Comment:
I'm a little confused by this comment. We built rsync support for (and with) SBGrid and they use the "file" store. Later we added the ability to use rsync/DCM with the "s3" store but I don't think any installation is actually running in that mode.

Reply:
This note dates from https://github.com/IQSS/dataverse/pull/6490 which was based on seeing the 's3' hardcoded entries from https://github.com/IQSS/dataverse/pull/5359/files in the code. Since I didn't know how to run rsync, I just maintained the dependence on s3 as the name and added the note.
At this point, I can't easily tell if the rsyncUploadSupported method is only called with the S3 variant (in which case the note is ~true but only w.r.t. to the S3 variant of rsync/DCM) or if it would also be called when using a file store (in which case the DatasetUtil.isAppropriateStorageDriver is probably incorrectly returning false and might either need to add the 'file' store or any store of type 'file' if there isn't a similar hardcode of the name 'file' somewhere.
In any case - this PR is just changing s3 from a string to a constant, so I'd suggest a separate issue if this should be pursued.

Reply:
Right. Looks like the comment was added in 6c506ab5 as part of #6488.

Let's not worry about this comment.

Obviously, rsync/DCM support is becoming a bit of a maintenance burden. If anyone is still using it, we should probably decide whether to breathe new life into it or rip it out.

Like you said, we can revisit all this in a future issue and PR.